tests/resources/expectations/models.test_gpt_bigcode.TestGPTBigCode.test_model_output

```diff
@@ -1 +1 @@
-0.0395454466342926,0.03477054834365845,0.026588499546051025,0.04722253978252411,0.0,0.04652555286884308,0.019217655062675476,0.0234697163105011,0.022567830979824066,0.033340491354465485,0.0022870004177093506,0.013856053352355957,0.010794378817081451,0.001753188669681549,0.023577645421028137,0.014053717255592346
\ No newline at end of file
+0.0360008180141449,0.04638493061065674,0.04776623845100403,0.04449990391731262,0.021900400519371033,0.03095899522304535,0.03315833956003189,0.0,0.037384480237960815,0.0542977899312973,0.051384031772613525,0.04188331961631775,0.006926089525222778,0.02746940404176712,0.03905876725912094,0.050389550626277924
```

Comment:
i would think that only changing fusion shouldn't change the model output? what happened here?

Reply:
I believe that the way we initialize weights is by sorting the keys and using a random seed. Because the weights are now different (names), we are going to get a different answer.

Reply:
One thing, I do have a test (locally) that checks weight conversion from old unfused weights to the new weights, however it requires the old model weights / signature be stored in resources. That could be added to this as well.

Reply:
just chiming in to say @JRosenkranz is correct here, the normal matrix initialization for the fused matrix will give different results and this is why this changes, not the fusion itself. If we were loading real weights into it, the result would be the same bar some FP16 matmul shenanigans